packages/gestalt/src/HelpButton.js

```diff
@@ -0,0 +1,200 @@
+
+  return (
+    <Flex alignItems="center" justifyContent="center" flex="none">
+      <Tooltip
+        text={accessibilityTooltipMessage}
+        idealDirection={idealDirection}
+        zIndex={tooltipZIndex}
+      >
+        <TapArea
+          accessibilityExpanded={isOpen}
+          accessibilityControls={popoverId}
+          accessibilityLabel={accessibilityTooltipMessage}
+          fullWidth={false}
+          onTap={onHandleTap}
+          ref={ref}
+          role="button"
+          rounding="circle"
+          onBlur={() => {
+            setFocused(false);
+            setActive(false);
+          }}
+          onFocus={() => setFocused(true)}
+          onMouseDown={() => setActive(true)}
+          onMouseEnter={() => setHovered(true)}
+          onMouseLeave={() => setHovered(false)}
+          onMouseUp={() => setActive(false)}
+          tabIndex={0}
+        >
+          <Box
+            width={16}
+            height={16}
+            rounding="circle"
+            display="flex"
+            alignItems="center"
+            justifyContent="center"
+            dangerouslySetInlineStyle={{
+              __style: {
+                backgroundColor: `var(--color-text-${bgIconColor})`,
+              },
+            }}
+          >
+            <Icon
+              accessibilityLabel={accessibilityIcon}
+              color="inverse"
+              icon="question-mark"
+              size={8}
+            />
+          </Box>
+        </TapArea>
+      </Tooltip>
+      {isOpen && (
+        <Box zIndex={zIndexLayer}>
+          <Popover
+            id={popoverId}
+            accessibilityLabel={accessibilityPopoverLabel}
+            anchor={ref.current}
+            onDismiss={() => {
+              toggleView();
+              setActive(false);
+            }}
+            idealDirection={idealDirection ?? 'down'}
+            positionRelativeToAnchor
+          >
+            <Box
+              padding={4}
+              rounding={4}
+              minWidth={230}
+              onBlur={() => {
+                setFocused(false);
+                if (!link?.href) {
+                  setIsOpen(false);
+                }
+              }}
+              onFocus={() => setFocused(true)}
+            >
+              <Box tabIndex={0}>{textElement}</Box>
+              {link?.href && (
+                <Box padding={3} width="100%" display="block" marginTop={3}>
+                  <Text weight="bold" size="300" align="center">
+                    <Link
+                      href={link?.href}
+                      externalLinkIcon="default"
+                      onClick={link?.onClick}
+                      onBlur={() => {
+                        setFocused(false);
+                        setIsOpen(false);
+                      }}
+                    >
+                      {link?.text ?? accessibilityDefaultLinkLabel}
```

Comment:
`accessibilityDefaultLinkLabel`

I think `accessibilityLinkLabel` should be required. So here engs could pass a href and get the Learn more. but learn more is also vague, it should be Learn more about XXXX. 

This is the web rotor, image all links saying Learn more.
![image](https://user-images.githubusercontent.com/10593890/218823751-bd14678e-bf84-4662-88c8-116347fdbf9b.png)

Reply:
I don't agree with a `pre label` on a11y, because this link should be an invite, "Visit our portal" or "Looking that deeply" so.
I defined `accessibilityLabel` as a required prop on link implementation, and serve an entire field available to our users.

Reply:
In case, i didnt communicate well.  

Link's text should be a required prop in the component, not optional. We shouldn't have a accessibilityDefaultLinkLabel to cover all cases. 

`Learn more` it's too generic and it's not accessible on a web rotor view. 

It should read `Learn more about something`.


If we leave here Learn more as default engineers aren't going to worry about the accessibilityLabel. Basically, if text is undefined, the accessibility label should be required. 

I feel it's less confusing if we require Text , which is the pattern we have in the whole library. 


The only new component with default text is the OverlayPanel confirmation modal, which it's less ambiguous than this test in this context.

Reply:
DONE